cypress/integration/core-opensearch-dashboards/opensearch-dashboards/apps/vis_type_vega/test_utils/utils.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import _ from 'lodash';
```

Comment:
consider only import the function needed, instead of import all

Reply:
Refactored imports